tiledbimg/converters/ome_zarr.py

```diff
@@ -115,12 +119,18 @@ def level_metadata(self, level: int) -> Dict[str, Any]:
         return {"pickled_zarrwriter_kwargs": pickle.dumps(writer_kwargs)}
 
     def metadata(self) -> Dict[str, Any]:
-        multiscale_meta = self.image.zarr.root_attrs.get("multiscales")
-        assert len(multiscale_meta) == 1
+        multiscales = self.image.zarr.root_attrs.get("multiscales")
+        assert len(multiscales) == 1, multiscales
+        multiscale = multiscales[0]
+        coordinate_transformations = (
```

Comment:
Why not with `get_coord = operator.itemgetter("coordinateTransformations")` and `list(map(get_coord, multiscale.get("datasets")))`

Reply:
To prevent KeyError in case `coordinateTransformations` is missing from a dataset dict

Reply:
According to NGFF spec here, I think we should error out in this cases but I leave it to your judgement.

> Each dictionary in "datasets" MUST contain the field "coordinateTransformations", which contains a list of transformations that map the data coordinates to the physical coordinates (as specified by "axes") for this resolution level.

Reply:
Let's ignore such dataset dicts for now since [ome-zarr-py](https://github.com/ome/ome-zarr-py/blob/master/ome_zarr/writer.py#L142) does the same.